globus_automate_client/queues_client.py

```diff
@@ -81,19 +84,19 @@ def update_queue(
         visibility_timeout: Optional[int] = None,
         **kwargs,
     ) -> Optional[GlobusHTTPResponse]:
-        body = dict(
-            id=queue_id,
-            label=label,
-            admins=admins,
-            senders=senders,
-            receivers=receivers,
-            delivery_timeout=delivery_timeout,
-            visibility_timeout=visibility_timeout,
-        )
+        body = {
```

Comment:
This is truly the more common method, but does anyone else kinda like the `dict(**kwargs)` syntax?

Reply:
I prefer dict literal syntax, though like all preferences this is just due to acclimation. :smiley: